ksql-engine/src/test/java/io/confluent/ksql/integration/IntegrationTestHarness.java

```diff
@@ -307,7 +307,7 @@ private Serializer getSerializer(Schema schema, DataSource.DataSourceSerDe dataS
                                                    DataSource.DataSourceSerDe dataSourceSerDe) {
     switch (dataSourceSerDe) {
       case JSON:
-        return new KsqlJsonDeserializer(schema);
+        return new KsqlJsonDeserializer(schema, false);
```

Comment:
There's a lot of lines changed in this file, yet all are white space changes except for this one.

This makes it hard to review - especially considering the size of this PR anyway.

Consider limiting whitespace changes, or submitting them under a different PR, or submitting them under a different commit in the same PR.

Reply:
I applied the Google style format here. There are still white spaces but at least this follows our standard style.